core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -807,11 +808,13 @@ void checkTaskPublishTimeout(TaskDef taskDef, Task task) {
         }
         // check the publishDuration is greater than TASK_PUBLISH_TIMEOUT_IN_SECONDS
         if (publishDuration > taskPublishTimeoutInMilliSeconds) {
+            TaskNotification taskNotification = new TaskNotification(task);
```

Comment:
Instead of TaskNotification, is there any other Task dto that is part of core module that can be used ?